now_common/utils.py

```diff
@@ -66,13 +70,35 @@ def setup_clip_music_apps(
         finetune_datasets=app_instance.finetune_datasets,
     )
 
+    gpu = '0'
+    device = 'cpu'
+    gpu_threshold = 250000
+    if 'NOW_CI_CPU' in os.environ:  # test gpu in CI if len(dataset) > 1000
```

Comment:
Did you mean `NOW_CI_RUN`?
```suggestion
    if 'NOW_CI_RUN' in os.environ:  # test gpu in CI if len(dataset) > 1000
```

Reply:
yep